base/reflection.jl

```diff
@@ -1106,6 +1112,69 @@ function methods(@nospecialize(f),
     return methods(f, Tuple{Vararg{Any}}, mod)
 end
 
+function _instancemethods(t::Type, @nospecialize(mt), lim::Int, world::UInt)
```

Comment:
This function is only used by `instancemethods` and always with the same arguments. Would it make sense to either specify those values as defaults for the corresponding arguments or inline the entire definition into `instancemethods`? For example, the former would look like
```suggestion
function _instancemethods(t::Type, @nospecialize(mt), lim::Int=-1,
                          world::UInt=get_world_counter())
```
and the latter would be (inside of `instancemethods`)
```julia
    tt = instance_signature_type(t, tmatch)
    for m in _methods_by_ftype(tt, -1, get_world_counter())::Vector
```